webdriver-spec.html

```diff
@@ -1435,288 +1435,532 @@
-   <dd><p>Set the proxy to No Proxy using implementation defined steps.
-    If this can not be set during this process return <a>error</a>.
+  <ol>
+  <li><p>Let <var>entry</var> be the entry in <var>secondary</var> at index
+  <var>k</var>.
 
-   <dt>"<code>autodetect</code>"
-   <dd><p>Set the proxy to Auto-Detect proxy setting from the network
-    using implementation defined steps.
-    If this can not be set during this process return <a>error</a>.
+  <li><p>If the name of <var>entry</var> is also the name of an entry in
+  <var>result</var>,
+  return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
 
-   <dt>"<code>system</code>"
-   <dd><p>Set the proxy to use system proxy settings using implementation defined steps.
-    If this can not be set during this process return <a>error</a>.
+  <li><p>Otherwise, add <var>entry</var> to <var>result</var>.
 
-   <dt>"<code>manual</code>"
-   <dd>
-    <ol>
-     <li><p>Let <var>ftp proxy</var> be the result of <a>getting a property</a>
-      named <code>ftpProxy</code> from <var>proxy</var>.
+  <li><p>Increment <var>k</var> by 1.
+  </ol>
 
-     <li><p>Let <var>ftp proxy port</var> be the result of <a>getting a property</a>
-      named <code>ftpProxyPort</code> from <var>proxy</var>.
+<li><p>Return <var>result</var>.
+</ol>
 
-     <li><p>Let <var>http proxy</var> be the result of <a>getting a property</a>
-      named <code>httpProxy</code> from <var>proxy</var>.
+<p>When <dfn>matching capabilities</dfn> with JSON Object argument
+  <var>capabilities</var>, an endpoint node <em>MUST</em> run the following
+  steps:
 
-     <li><p>Let <var>http proxy port</var> be the result of <a>getting a property</a>
-      named <code>httpProxyPort</code> from <var>proxy</var>.
+<ol>
+<li><p>Let <var>matched capabilities</var> be a JSON Object with the following
+entries:
 
-     <li><p>Let <var>ssl proxy</var> be the result of <a>getting a property</a>
-      named <code>sslProxy</code> from <var>proxy</var>.
+  <dl>
+  <dt>"<code>browserName</code>"
+  <dd>Lowercase name of the user agent.
 
-     <li><p>Let <var>ssl proxy port</var> be the result of <a>getting a property</a>
-      named <code>sslProxyPort</code> from <var>proxy</var>.
+  <dt>"<code>browserVersion</code>"
+  <dd>The user agent version, as a string.
 
-     <li><p>Let <var>socks proxy</var> be the result of <a>getting a property</a>
-      named <code>socksProxy</code> from <var>proxy</var>.
+  <dt>"<code>platformName</code>"
+  <dd>Lowercase name of the current platform.
 
-     <li><p>Let <var>socks proxy port</var> be the result of <a>getting a property</a>
-      named <code>socksProxyPort</code> from <var>proxy</var>.
+  <dt>"<code>platformVersion</code>"
+  <dd>The platform version, as a string.
 
-     <li><p>If <var>socks proxy</var> and <var>socks proxy port</var> is defined:
+  <dt>"<code>acceptSslCerts</code>"
+  <dd>Boolean false, indicating the session <em>will not</em> implicitly trust
```

Comment:
Boolean true, since the largest audience for this spec are QA engineers who may not be able to control their environment enough to be able to have a valid SSL cert, and who also are most likely to be inexperienced enough developers to not be able to set this parameter. *sigh*